internal/gengapic/example.go

```diff
@@ -85,7 +98,14 @@ func (g *generator) exampleMethod(pkgName, servName string, m *descriptor.Method
 		return err
 	}
 
-	g.exampleInitClient(pkgName, servName)
+	// Pick the first transport for simplicity. We don't need examples
+	// of each method for both transports when they have the same surface.
```

Comment:
Where do these examples go? If they go into `_test.go` files, then they show up in godoc alongside the init methods, right? In that case, it might be worthwhile to have one for each transport, even if the only difference is one name. WDYT?

Reply:
They go into the `_example_test.go` files which are indeed included in the GoDoc, but there is only 1 exported client type, the transport agnostic one. Thus, there will only be docs for that type's methods. There is no difference in the usage of a method based on transport and the only difference is in the factory method used to create the client, which will have separate examples because they are separate publicly exported methods. It would be a one line difference but double the examples, and I don't think that's all that valuable.

Reply:
ok, sg

Reply:
One very minor idea for clients that have multiple transports: in the examples for non-factory methods, in these init lines, maybe we continue having only one transport but add a comment ` // use the Init method corresponding to your desired transport`. That way we avoid code duplication but also don't confuse people who may want to use the transport not cited in the example.

Very minor thing, we  don't need to do it now, and we can discuss.